tests/helm_chart_imagepullsecrets/helm_chart_imagepullsecrets_suite_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright DataStax, Inc.
```

Comment:
This test will probably be removed from the PR soon, because any of the tests can be used to test the custom image pull secret.